### PR TITLE
Activity Selection in C

### PR DESCRIPTION
**Pull request by yashaswibiyahut**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Issue no: #826
Activity selection in c.
**Approach :** Greedy Algorithm.
**Time Complexity -**
The complexity of this problem is O(n log n) when the list is not sorted.
When the sorted list is provided the complexity will be O(n).

_Below is the output of given code._

### Checklist

- [x] I've read the contribution guidelines.
- [x]  I've checked the issue list before deciding what to submit.
- [x]  I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
activity
Output of above code

**Comment by yashaswibiyahut** [comment]
kindly review this PR @Kajol-Kumari @SKAUL05 @siddharth25pandey @iamrajiv @HarshCasper
I made all the changes as per the requirement in my last PR.

File: C/cp/Activity_selection.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			if(finish[j]>finish[j+1])
+			{
+				//call swap function
+				//pass by address
+				swap(&finish[j],&finish[j+1]);
+				swap(&start[j],&start[j+1]);
+			}
+		}
+	}
+	//condition for Activity selection
```
Comment (Kajol-Kumari) [inline C/cp/Activity_selection.c]: Please use multi line comment here.

File: C/cp/Activity_selection.c [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	for(i=0;i<num;i++)
+	{
+		scanf("%d",&start[i]);
+		scanf("%d",&finish[i]);
+	}
+	//sorting starts here
+	//Bubble sort - compare adjacent element and swap it
+	for(i=0;i<num;i++)
+	{
+		//In bubble sort,
```
Comment (Kajol-Kumari) [inline C/cp/Activity_selection.c]: same, multi line comment

**Review by Kajol-Kumari — COMMENTED** [review]
@yashaswibiyahut minor changes required, rest LGTM :+1:

File: C/cp/Activity_selection.c (hunk 1 above)
Comment (yashaswibiyahut) [reply C/cp/Activity_selection.c]: done @Kajol-Kumari  kindly review it.

File: C/cp/Activity_selection.c (hunk 2 above)
Comment (yashaswibiyahut) [reply C/cp/Activity_selection.c]: done @Kajol-Kumari  kindly review it

**Review by iamrajiv — APPROVED** [review]

**Comment by yashaswibiyahut** [comment]
kindly review this pr @Kajol-Kumari @siddharth25pandey

**Comment by siddharth25pandey** [comment]
@yashaswibiyahut How you are using **Malloc** without including **stdlib** header.
Can you explain me briefly.
Thanks!

**Comment by yashaswibiyahut** [comment]
@siddharth25pandey  Its my mistake.
I make changes on my IDE at the time of compilation but forgot to update here.
To use the malloc() & calloc() functions, the standard header file “stdlib.h" must be present inside the code.
kindly review this.

**Comment by siddharth25pandey** [comment]
@yashaswibiyahut Approving this **PR** but don't do these type of silly mistakes in future, it's my personal suggestion

**Review by siddharth25pandey — APPROVED** [review]

**Comment by yashaswibiyahut** [comment]
Sure I will take care of this from the next time @siddharth25pandey 
Thanks!

**Comment by yashaswibiyahut** [comment]
@Kajol-Kumari  kindly review this PR

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by yashaswibiyahut** [comment]
Kindly merge this PR @siddharth25pandey @HarshCasper

**Pull request merged**